Assets/Scripts/WeaponInventory.cs

```diff
@@ -53,8 +61,16 @@ public void EquipWeapon(WeaponId weaponId)
             // Set ammo to infinite for now
             switch (weaponId)
             {
+                case WeaponId.VETO:

```

Comment:
another argument for this is that it doesn't make sense to have each individual player's weapon inventory have their own member variable for what is essentially a set of constants.